src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -24,4 +34,56 @@ public void verifyContentUnderCertifiedHealthITProductListingAndURL(final String
         assertEquals(actualText, text);
         assertEquals(actualURL, urlLink);
     }
+
+    /**
+     * Assert that title should be CHPL API.
+     * @param title expected as CHPL API
+     */
+    @Then("^CHPL API page title should be \"([^\"]*)\"$")
+    public void chplApiPageTitleShouldBe(final String title) {
+        String chplAPITitle = getDriver().getTitle();
+        Assert.assertEquals(chplAPITitle, title);
+    }
+
+    /**
+     * Click all the controller elements link.
+     * @param arg1 as controller elements
+     */
+    @When("^I click on \"([^\"]*)\" link$")
+    public void iClickOnLink(final String arg1) {
+        String xPath = "//*[text()='" + arg1 + "']";
+        WebElement link = ChplAPIPage.controllerElementName(getDriver(), xPath);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
+    }
+
+    /**
+     * Assert that contents from webpage matches the contents of excel.
+     * @param str as sheet name in the excel
+     */
+    @Then("^list of \"([^\"]*)\" operations should be displayed without deprecated calls$")
+    public void listOfOperationsShouldBeDisplayed(final String str) {
+        String sheetName = null;
+        WebDriver driver = getDriver();
+        // Get element from the webpage into the list
+        List<WebElement> listWebElements = ChplAPIPage.controllerElementList(driver);
+        if (str.equalsIgnoreCase(CORR_ACT_PLN_CTRL)) {
```

Comment:
The corrective-action-plan controller is entirely deprecated, and probably doesn't need to be here at all